cmd/admin.go

```diff
@@ -253,6 +254,33 @@ var (
 		Action: runAddOauth,
 		Flags:  oauthCLIFlags,
 	}
+
+	subcmdSendMail = cli.Command{
+		Name:   "sendmail",
+		Usage:  "Send a message to all users",
+		Action: runSendMail,
+		Flags: []cli.Flag{
+			cli.StringFlag{
+				Name:  "title",
+				Usage: `a title of a message`,
+				Value: "",
+			},
+			cli.StringFlag{
+				Name:  "content",
+				Usage: "a content of a message",
+				Value: "",
+			},
+			cli.BoolFlag{
+				Name:  "force,f",
+				Usage: "A flag to bypass a confirmation step",
+			},
+			cli.DurationFlag{
+				Name:  "timeout",
+				Usage: "A timeout of sending emails",
+				Value: 0,
```

Comment:
i guess this are seconds?

Reply:
If you are speaking about default value It's not actually, as I've seen how mail service works when you set a Duration value to 0. There will be not timeout tracker. So it set intentionally to 0

Reply:
If you're about an expected format it uses a Duration format so you can specify it via prefixes like `3s`, `10m`, `1h` etc.